lib/cluster.js

```diff
@@ -198,7 +198,9 @@ Cluster.prototype.fetchStats = function fetchStats(callback) {
             foundMembers[memberAddr] = true;
             fetchQueue.push(memberAddr, function onStatsFetched(err, stats) {
                 if (err) {
-                    console.error('Error while fetching node stats:', err);
+                    // We don't want to handle this error loudly. It happens
```

Comment:
I'd rather not oscillate between reporting errors blindly and swallowing them completely. Let's do better.

Can we collect the amount of errors that have occurred per issued command and report that to the console in a consistent way?